app/package.json

```diff
@@ -23,6 +23,7 @@
     "@mui/material": "^5.2.3",
     "@reduxjs/toolkit": "^1.7.0",
     "@tippyjs/react": "^4.2.6",
+    "@toruslabs/torus-embed": "^1.22.1",
```

Comment:
looks like you added this manually because I don't see a package.lock

Try removing this, saving the file, then running (from repo root)
`npm install -w app @toruslabs/torus-embed`